docs/configuration.md

```diff
@@ -95,7 +95,7 @@ $ neofs-s3-gw --healthcheck_timeout 15s --connect_timeout 1m --rebalance_interva
 ### Monitoring and metrics
```

Comment:
I think in `--help` and in line 49:
```
### Listening on address and TLS
You can make the gateway listen on specific address using the `--listen_address` option
```
we should mention that only one address can be set in this way. Or make it similar to `--peers`, but I'm not sure.